RoutesApp_ios/repository/models/remote/Lines.swift

```diff
@@ -34,7 +36,8 @@ struct Lines: Codable {
         entity.idCategory = idCategory
         entity.name = name
         entity.category = categoryFirst ?? LineCategoryEntity()
-        entity.createdAt = Date()
+        entity.createAt = Date()
+        entity.updateAt = Date()
```

Comment:
If we are using the same value for both properties, would it be better if we keep just one of then?

Reply:
Same for other entities

Reply:
my bad i forgot to changed it, done